### DISCUSSION TITLE
base_url setting

### DISCUSSION THREAD

**Discussion by jhyungjeon** [jupyter/notebook#7091] [Q&A]
hello. 
The base_url variable in the Jupyter notebook appears to be fixed to "/".
In order to change the variable, which source should I edit it from? 
I am curious about the part that sets the default value of the essential base_url rather than the base_url set in jupyter_server_config.py.

**Comment by jtpio** [comment]
`/` is the default, but it's configurable for example with `--ServerApp.base_url=/custom_base_url`

**Reply by jhyungjeon** [reply]
Thanks for your reply. I have jupyter_server_config.py. I was asking about something other than the way it is set up. I found the answer. There was a setting in serverapp.py so I fixed it.
